b2/sync/report.py

```diff
@@ -198,6 +198,9 @@ def local_permission_error(self, path):
             'WARNING: %s could not be accessed (no permissions to read?)' % (path,)
         )
 
+    def local_symlink_skip_notice(self, path):
```

Comment:
event (non-progress) method names in this class are an answer to a question "what happened?", so "local access error", "local permission error", and here it should be "symlink skipped". Furthermore, in default implementation I think the method should be just `pass`: if the user specified an option to skip all symlinks, he doesn't expect to see names of all of those symlinks on his console.